### DISCUSSION TITLE
Maker Day

### DISCUSSION THREAD

**Discussion by aetkie** [UNC-Project-Open-AAC/OS-DPI#239] [General]
Did you embark on an AAC journey at Maker Day? Show off what you made in this post. 

Screenshot the UI and post it below in a comment

**_-or-_**

Go to **File** --> **Export**
Save the file
Upload the .osdpi file in a comment below

**Comment by cinderdazzle** [comment]
Hello. I know this was for ATIA 2024, and it almost time for ATIA 2025 so you may have other things to think about. 
I was trying to work through the Keyword Word Prediction. I got all the keys like shift for changing cases, delete, and speak working.  I could not get the prediction working. I tried to make it look just like the tutorial, but I kept getting an error on  **replace_last(#label + ‘ ‘)**
Also, I'm not quite sure why I have lots of extra space left over in the whole display, but that was secondary to getting prediction to work. Thank you for the tutorial. I plan to work through them and read the documents to see if I can learn what I need to to test a design idea out. 

<img width="1335" alt="Screenshot 2025-01-11 at 8 28 48 PM" src="https://github.com/user-attachments/assets/d95f4ecc-9597-49f2-ba03-9a55902ec5fc" />

**Reply by gbishop** [reply]
I think you are using the "grave accent" (below tilde in the upper right of my keyboard) instead of the "single quote" below the double quote on the right near enter.

Also, you have misspelled $Display as $Dispaly

**Reply by cinderdazzle** [reply]
Thank you! I was able to fix these errors. Thank you for catching my mistake on display. For the other error, I typed it as written in the tutorial instead of copying and pasting. Then I found another error not pictured which fixed my word prediction issue. This was fun.

**Reply by gbishop** [reply]
Great! Congratulations. It feels really great when these things finally work,

**Reply by aetkie** [reply]
@cinderdazzle thanks for giving this tutorial a try and sharing your progress!
